src/main/resources/application-dev.yml

```diff
@@ -6,26 +6,18 @@ spring:
     compose:
       file: compose-dev.yaml
 
-  datasource:
-    driver-class-name: org.postgresql.Driver
```

Comment:
Janusz, czemu obsługa bazy danych z profilu dev została usunięta i dodana do application.yml?

Reply:
Te zmiany wynikały z konfliktu konfiguracji tych samym propertitesów na różnych profilach (dev i test). Przeniosłem to do appliation.yml żeby zminimalizować źródło generowanych problemów. Teraz po wyeliminowaniu tych problemów te ustawienia mogą wrócić do swoich profilowych propertiesów tzn application-dev oraz application-test na co dodaję osobnego commita